posydon/popsyn/population_params_default.ini

```diff
@@ -116,6 +116,12 @@
     # True False
   verbose = False
     # True False
+  
+[step_disrupted]
+  import = ['posydon.binary_evol.DT.step_disrupted','disrupted_step']
+  import = ['posydon.binary_evol.DT.step_isolated',isolated_step']
```

Comment:
This line should be deleted, as one should only import one step at a time. Also, does this step have inputs? Does it take the same one as of the detached step? If yes, comment here in needed, else you should define them both in the step initialisation and here.

From the changes done in the flow chart, the isolated_step is not part of the default setup, right?